src/views/SignUp/SignUp.js

```diff
@@ -0,0 +1,105 @@
+import React,{Component} from "react";
+import Layout from '../../components/Layout';
+import camelcase from 'camelcase';
+import serverApi from '../../api/worker';
```

Comment:
Always name the imports as the filename.
```suggestion
import worker from '../../api/worker';
```